product-create-search/src/Controller/PaintingController.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Controller;
+
+use App\BusinessLogic\CrudMysqLI;
+use App\BusinessLogic\SendToKafkaI;
+use App\Events\KafkaEvent;
+use App\Listeners\KafkaListenerI;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+
+class PaintingController extends AbstractController
```

Comment:
We should create our Base Controller , and all Controllers should extend it , 
We do not want to be coupled to the framework